src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xdecoder/common/ErrorMessage.java

```diff
@@ -0,0 +1,36 @@
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.xdecoder.common;
+
+import java.util.HashMap;
+import java.util.Optional;
+
+/**
+ * Error message
+ *
+ * @author Harry / Symphony Dev Team<br>
+ * Created on 5/15/2022
+ * @since 1.0.0
+ */
+public class ErrorMessage {
+	private ErrorMessage() {
+	}
+
+	private static HashMap<String, String> errorMessages;
+
+	/**
+	 * This method is used to convert error message to specify error message
+	 * @param errorMessage error message that want to convert
+	 * @return String converted error message
+	 */
+	public static String convertErrorMessage(String errorMessage) {
+		if (errorMessages == null) {
+			errorMessages = new HashMap<>();
+			errorMessages.put("Decoder 2 configuration failed: \\\"Stream still in use\\\"\"}", "Please choose a different secondary stream");
+		}
+		Optional<String> result = Optional.ofNullable(errorMessages.get(errorMessage));
+		if (result.isPresent()) {
```

Comment:
Convert this if to return result.orElse(errorMessage);

Reply:
I fixed it